### DISCUSSION TITLE
No p element in scope but a p end tag seen

### DISCUSSION THREAD

**Discussion by Bikersguide** [chipzoller/hugo-clarity#440] [General]
After checking my website with the W3C validator, I get error messages for posts that contain images.

Error: No p element in scope but a p end tag seen

The error can be easily fixed. I edited the render-image.html and added a `</p>` at the beginning and a `<p>` at the end.
With this change the error is gone.

Maybe it will help someone!

**Comment by Tekki** [comment]
I get the same error, the cause seems to be the `figure` tag nested inside `p`. Does anyone know the reason for that?

**Comment by DjSlash** [comment]
This is Hugo behaviour. Since [v0.108.0](https://github.com/gohugoio/hugo/releases/tag/v0.108.0) they added a option `markup.goldmark.parser.wrapStandAloneImageWithinParagraph`, when set to `false` it gets rid of the paragraph tags.
